proto/buf/alpha/registry/v1alpha1/sync.proto

```diff
@@ -82,3 +82,11 @@ message SyncGitCommitResponse {
   // SyncPoint is the latest syncpoint for the SyncGitCommit request.
   GitSyncPoint sync_point = 1;
 }
+
+// Branch holds branch information when pushed as a sync git point.
+message Branch {
+  // Name of the branch.
+  string name = 1;
+  // Is this branch the default branch in the repo.
+  bool is_repo_default = 2;
+}
```

Comment:
@doriable @stefanvanburen I'm not 100% this is required. This is so the BSR knows which commits should be marked as "HEAD" label. Am I correct assuming that namespace label will only apply to the default repo branch?